static/js/components/UserChip.js

```diff
@@ -18,7 +18,6 @@ const UserChip = ({ profile }: {profile: Profile}): React$Element<*> => (
         { mstr(getEmployer(profile)) }
       </span>
       <a href={`/users/${profile.username}`} className="mm-minor-action">
-        <Icon name="person" />
```

Comment:
Can you remove line 4 in this file since we no longer need `Icon` after your change?